packages/core/src/amazonq/webview/ui/tabs/constants.ts

```diff
@@ -24,7 +24,7 @@ export const TabTypeDataMap: Record<TabType, TabTypeData> = {
     featuredev: {
         title: 'Q - Dev',
         placeholder: 'Describe your task or issue in as much detail as possible',
-        welcome: `Hi, I'm the Amazon Q Developer Agent for software development.
+        welcome: `Welcome to the agent for software development.
```

Comment:
Why we change this welcome message back? Did you rebase the origin/main?

Reply:
Added it back your version, sounds like its more updated than mine lol